test/vm_tests/transformer/impl_generation_test.dart

```diff
@@ -382,32 +382,60 @@ main() {
         });
       });
 
-      test('props mixins', () {
-        preservedLineNumbersTest('''
-          @PropsMixin() class FooPropsMixin {
-            Map get props;
+      group('for props mixins', () {
+        test('without type parameters', () {
+          preservedLineNumbersTest('''
+            @PropsMixin() class FooPropsMixin {
+              Map get props;
 
-            var bar;
-            var baz;
-          }
-        ''');
+              List<T> bar;
+              var baz;
+            }
+          ''');
 
-        var transformedSource = transformedFile.getTransformedText();
-        expect(transformedSource, contains('abstract class \$FooPropsMixin {}'));
+          expect(transformedFile.getTransformedText(), contains('abstract class \$FooPropsMixin {}'));
+        });
+
+        test('with type parameters', () {
+          preservedLineNumbersTest('''
+            @PropsMixin() class FooPropsMixin<T> {
```

Comment:
Just to be safe, let's use two params and have one of them use a bound as well to make sure that all works:
```
FooPropsMixin<T extends Iterable<T>, U>
```